src/controllers/mouseoverTextController.js

```diff
@@ -1,43 +1,48 @@
-const mouseoverTextController = (function (MouseoverText) {
-    const createMouseoverText = function (req, res) {
-        const newMouseoverText = new MouseoverText();
-        newMouseoverText.mouseoverText = req.body.newMouseoverText;
-        newMouseoverText.save().then(() => res.status(201).json({
-            _serverMessage: 'MouseoverText succesfuly created!',
-            mouseoverText: newMouseoverText,
-        })).catch(err => res.status(500).send({ err }));
-    };
+const mouseoverTextController = function (MouseoverText) {
+  const createMouseoverText = function (req, res) {
+    const newMouseoverText = new MouseoverText();
+    newMouseoverText.mouseoverText = req.body.newMouseoverText;
+    newMouseoverText
+      .save()
+      .then((result) =>
+        res
+          .status(201)
+          .send({ _serverMessage: 'MouseoverText successfully created!', mouseoverText: result }),
+      )
+      .catch((error) => res.status(500).send(error));
+  };
 
-    const getMouseoverText = function (req, res) {
-        MouseoverText.find()
-            .then(results => res.status(200).send(results))
-            .catch(error => res.status(404).send(error));
-    };
+  const getMouseoverText = function (req, res) {
+    MouseoverText.find()
+      .then((results) => res.status(200).send(results))
+      .catch((error) => res.status(404).send(error));
+  };
 
-    const updateMouseoverText = function (req, res) {
-        // if (req.body.requestor.role !== 'Owner') {
-        //     res.status(403).send('You are not authorized to update mouseoverText!');
-        // }
-        const { id } = req.params;
+  const updateMouseoverText = function (req, res) {
+    // if (req.body.requestor.role !== 'Owner') {
+    //     res.status(403).send('You are not authorized to update mouseoverText!');
+    // }
+    const { id } = req.params;
 
-        return MouseoverText.findById(id, (error, mouseoverText) => {
-            if (error || mouseoverText === null) {
-                res.status(500).send('MouseoverText not found with the given ID');
-                return;
-            }
+    return MouseoverText.findById(id, (error, mouseoverText) => {
+      if (error || mouseoverText === null) {
+        res.status(500).send({ error: 'MouseoverText not found with the given ID' });
```

Comment:
Same here, the `send` argument has changed from `'MouseoverText not found with the given ID'` to `{ error: 'MouseoverText not found with the given ID' }`. Was this update required? If so, was it tested with the frontend?